### ISSUE TITLE
problem displaying delays 

### ISSUE THREAD

**Issue by wimThoelke** [KristjanESPERANTO/MMM-PublicTransportHafas#27]
**Bug description**
non-breaking spaces ("\&nbsp;") are not decoded correctly (see screnshot).

**Module config**

```
{
   module: "MMM-PublicTransportHafas",
         position: "bottom_center",
            config: {
            tableClass: "xsmall",
            hafasProfile: "db",
            timeInFuture:  60,
            stationID: "*******",                   
            stationName: "Bahnhof ********", 
            direction: "",                    
            excludedTransportationTypes: [],  
            ignoredLines: [],                 
            timeToStation: 5,                
             // Look and Feel
            displayLastUpdate: true,           
            maxUnreachableDepartures: 0, 
            maxReachableDepartures: 7,     
            showColoredLineSymbols: true,
            customLineStyles: "",           
            showOnlyLineNumbers: false, 
            showTableHeadersAsSymbols: true,
            useColorForRealtimeInfo: true  
          }
      },
```
      
 

**Screenshots**
![grafik](https://user-images.githubusercontent.com/96778370/147567784-1f95174c-f693-4b92-becc-9889c258adee.png)

**Additional context**
none.

**Comment by KristjanESPERANTO** [comment]
Oh, that happened because [I replaced innerHTML with innerText](https://github.com/KristjanESPERANTO/MMM-PublicTransportHafas/commit/aad9a1d1b0f5ae0f9d9c4a3da5d594aa52365d02). I didn't notice the effect because I tested with `showAbsoluteTime: false`.

So I replaced the non-breaking spaces with CSS padding, it should look better now :slightly_smiling_face: 

Thanks for the good bug description, so I quickly knew where to start.
